tests/modules/asset_groups/resources/test_permissions.py

```diff
@@ -107,10 +103,8 @@ def test_create_patch_asset_group(
         asset_group_utils.delete_asset_group(
             flask_app_client, researcher_1, asset_group_guid
         )
-        try:
-            delete_remote(str(temp_asset_group.guid))
-        except GitlabInitializationError:
-            pass
+        # temp_asset_group should be already deleted on gitlab
+        assert not AssetGroup.is_on_remote(str(temp_asset_group.guid))
```

Comment:
@hwindsor I've added this assertion to make sure the gitlab project is already deleted